cmake/CeleritasLibrary.cmake

```diff
@@ -543,123 +573,53 @@ endfunction()
 # the 3 libraries (static, middle, final) libraries needed
 # for a separatable CUDA library
 function(celeritas_target_link_libraries target)
-  if(NOT CELERITAS_USE_CUDA)
+  if(NOT CELERITAS_USE_VecGeom OR NOT CELERITAS_USE_CUDA)
     target_link_libraries(${ARGV})
-  else()
-    celeritas_strip_alias(target ${target})
-
-    celeritas_lib_contains_cuda(_contains_cuda ${target})
-
-    set(_target_final ${target})
-    set(_target_middle ${target})
-    if (_contains_cuda)
-      get_target_property(_targettype ${target} CELERITAS_CUDA_LIBRARY_TYPE)
-      if(_targettype)
-        get_target_property(_target_final ${target} CELERITAS_CUDA_FINAL_LIBRARY)
-        get_target_property(_target_middle ${target} CELERITAS_CUDA_MIDDLE_LIBRARY)
-        get_target_property(_target_object ${target} CELERITAS_CUDA_OBJECT_LIBRARY)
-      endif()
-    endif()
+    return()
+  endif()
```

Comment:
Note that all subsequent changes are just due to indentation from returning early (rather than putting everything in an ELSE).